charts/kong-operator/values.yaml

```diff
@@ -0,0 +1,114 @@
+image:
+  repository: docker.io/kong/gateway-operator
+  tag: "1.5"
```

Comment:
Should we use `1.6` as the default valuie here?

Reply:
Agreed, I forgot to update this after 1.6 has been released. 5ec39ad8ba4bc2dc05fd04f3ff435ec6b75d0bb8 covers that.